### ISSUE TITLE
Disabled buttons(Action buttons) are not accessible

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#41195]
### Steps to reproduce the issue
Thanks @brianteeman for this finding. 

Disabled buttons with listcheck=true such as the Actions button in our backend list views are not WCAG 2.1.
In this case we set the button to disabled until an item is selected.
The a11y issue with this is that it makes the button invisible to AT so the user will not even know it exists - it has no focus at all.

The alternative is to use **aria-disabled** instead. 
That keeps the button as focussable and announces that the button is disabled (and optionaly what you have to do to make it enabled). 

### Expected result
Also the disabled button is focussable and accessible

### Actual result
Not accessible

**Comment by chmst** [comment]
Closing, as #47324 resolves the issue. 
Thank you @adarshdubey03
